readme.md

```diff
@@ -760,6 +760,13 @@ Default: `Infinity`
 
 The maximum amount of items that should be emitted.
 
+###### pagination.requestLimit
+
+Type: `number`\
+Default: `100`
```

Comment:
Why `100`? If this is just meant to prevent a mistake in development, I think the default should be higher. Maybe just `Infinity`. Then you just set it if you need it.

Reply:
Okay, I increased it to 10000. Defaulting to `Infinity` is too much I think as also at runtime in production you could trigger accidentally an infinite amount.